src/Nodes/InterfaceLabelNode.ts

```diff
@@ -34,10 +34,14 @@ export class InterfaceLabelNode implements INode {
                 "get",
             ))).data;
             TelemetryClient.sendEvent(Constants.IoTHubAILoadInterfacesTreeDoneEvent, { Result: "Success" });
-            if (!interfaces || !interfaces.interfaces || Object.keys(interfaces.interfaces).length === 0) {
+            let interfaceIds = [];
+            if (Utility.getReportedInterfacesFromDigitalTwin(interfaces)) {

```

Comment:
getReportedIntefacesFromDigitalTwin method can return the interfaces values? So there no need to to Object.values(...) again?

Reply:
Updated